cypress/e2e/zenscrape/documentation.cy.js

```diff
@@ -0,0 +1,112 @@
+describe('Zenscrape documentation page', () => {
+
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // Returning false here prevents Cypress from failing the test
+        return false;
+    });
+
+    before(() => {
+        cy.visit('https://zenscrape.com/');
+    });
+
+    it('1. Test to accept cookies if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to click the documentation hyperlink', () => {
+        cy.get('.gap-8 > [href="//app.zenscrape.com/documentation"]').click();
+    });
+
+    it('3. Test to check the text in the documentation section', () => {
+        cy.wait(8000);
```

Comment:
why wait?

Reply:
can you try cy.document().its('readyState').should('eq', 'complete');

Reply:
Removed the cy.wait and it's working fine